platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/rest/ApplicationUploadEndpoint.java

```diff
@@ -233,10 +234,15 @@ private File createFileFromAttachement(Attachment attachment, Response response)
     /**
      * Setter method for DEFAULT_FILE_LOCATION for testing purposes
      *
-     * @param fileLocation
-     *            the desired fileLocation
+     * @param fileLocation the desired fileLocation
      */
     void setDefaultFileLocation(String fileLocation) {
-        defaultFileLocation = fileLocation;
+
+        if (!Paths.get(fileLocation)
```

Comment:
Same comment re: using `Paths.get` or `File.separator`.

Also, I've already seen a similar/identical method once in this PR and expect to see it again. You should look to extract it to a common utility method.